vividus-allure-adaptor/src/main/java/org/vividus/bdd/report/allure/VividusLabel.java

```diff
@@ -68,22 +75,32 @@ String getMetaName()
         return metaName;
     }
 
-    public Set<String> extractMetaValues(Meta storyMeta, Meta scenarioMeta)
+    public Set<Entry<String, String>> extractMetaValues(Meta storyMeta, Meta scenarioMeta)
     {
         return Stream.of(storyMeta, scenarioMeta)
                 .map(MetaWrapper::new)
-                .map(meta -> meta.getPropertyValues(metaName))
+                .map(meta -> meta.getPropertiesByKey(k -> k.startsWith(metaName)).entrySet())
                 .flatMap(Collection::stream)
+                .flatMap(e -> Stream.of(StringUtils.split(e.getValue(), ";"))
```

Comment:
please, use `org.vividus.bdd.model.MetaWrapper#META_VALUES_SEPARATOR` instead of `;`